uint8 mobility
uint32[] freeHands
uint32[] hasObjects
string[] skeletonNames
Joint[] skeletonJoint
Entity meEntity

